app/src/components/report/Table.tsx

```diff
@@ -0,0 +1,164 @@
+    paddingBottom: '10px',
+};
+
+type TableProps = {
+    rows: IReport[];
+};
+
+export default function Table({ rows }: TableProps) {
+    const [clickedIndex, setClickedIndex] = useState(-1);
+
+    const columns: GridColDef[] = [
+        {
+            field: 'reportId',
+            headerName: '',
+            width: 80,
+            renderCell: (cellValues: GridRenderCellParams<any>) => {
+                return (
+                    <IconButton
+                        aria-label={
+                            cellValues.value === clickedIndex ? 'Collapse row' : 'Expand row'
+                        }
+                        onClick={() => {
+                            clickedIndex === cellValues.value
+                                ? setClickedIndex(-1)
+                                : setClickedIndex(cellValues.value);
+                        }}
+                    >
+                        {cellValues.value === clickedIndex ? (
+                            <ExpandLessIcon />
+                        ) : (
+                            <ExpandMoreIcon />
+                        )}
+                    </IconButton>
+                );
+            },
+        },
+        {
+            field: 'name',
+            headerName: 'Name',
+            width: 237,
+            renderCell: (cellValues: GridRenderCellParams<any>) => {
+                return (
+                    <Box>
+                        <div>
+                            {/* First row item, ex: the name will be displayed here */}
+                            {cellValues.value ? cellValues.value : 'Anonymous'}
+                            <Collapse
+                                in={cellValues.id === clickedIndex}
+                                aria-expanded={cellValues.value === clickedIndex}
+                            >
+                                <Box sx={expandedRowStyle}>
+                                    {/* Expanded row item */}
+                                    {cellValues.row.phoneNumber}
+                                    <br />
+                                    {cellValues.row.emailAddress}
+                                </Box>
+                            </Collapse>
+                        </div>
+                    </Box>
+                );
+            },
+        },
+        {
+            field: 'reportCategory',
+            headerName: 'Issues',
+            //TODO: change this to something else, because this is an array of issues
+            type: 'singleSelect',
+            valueOptions: ['Illegal dump', 'Clogged drains', 'Other'],
+            width: 237,
+            renderCell: (cellValues: GridRenderCellParams<any>) => {
+                return (
+                    <Box>
+                        <div>
+                            {/* First row item */}
+                            {cellValues.value.map((issue: any) => (
+                                <p style={{margin:0}}>{issue}</p>
```

Comment:
Nit: should be styled as `{{ margin: 0 }}`

(install prettier in your IDE for auto formatting)